backend/open_webui/routers/openai.py

```
@@ -883,6 +887,88 @@ async def generate_chat_completion(
+        r.raise_for_status()
+        if "text/event-stream" in r.headers.get("Content-Type", ""):
+            streaming = True
+            return StreamingResponse(
+                r.content,
+                status_code=r.status,
+                headers=dict(r.headers),
+                background=BackgroundTask(
+                    cleanup_response, response=r, session=session
+                ),
+            )
+        else:
```

Ruff (0.11.9)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)